### PR TITLE
splash page looking good. May have to adjust some positions when navb…

### LINKED ISSUE #2: Homepage styling
- [x] `/home` is fully styled, according to the wireframe
- [x] image paths

Testing.

### LINKED ISSUE #5: F0: User Auth: redirect
### As an unauthenticated user, I am redirected to the splash / login / signup page
  - [x] Unauthenticated visits are redirected to `/home`
  - [x] Authenticated visits are NOT redirected to `/home`

### LINKED ISSUE #6: F0: User Auth: Demo User
### As any user, I can use the Demo User to try out the site.
  - [x] There is a demo user button
  - [x] Clicking the demo user button logs the user into the demo user account

### LINKED ISSUE #7: F0: User Auth: Login
### As a user, I can login to my account, so that I can access my boards, lists, and cards
  - [x] Login Button Displays Login / Signup Modal
  - [x] Successful login redirects user to `/dashboard`
  - [x] Unsuccessful login displays error message

### LINKED ISSUE #8: F0: User Auth: Logout
### As a user, I can logout.
  - [x] Logout button displays for logged-in users
  - [x] Clicking logout logs out user and redirects to `/home`

### PR DESCRIPTION
**Pull request by jonevanmoore**
splash page looking good. May have to adjust some positions when navbar is removed
Screen Shot 2022-05-03 at 12 41 27 PM
d

### CODE REVIEW

File: frontend/src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -3,20 +3,53 @@ import './LandingPage.css'
...
     return (
         <div className="home-body-div">
-            <div className="home-top-div">
-                <div className="logo-div">
-                    <img src='/logo-dark.png' className='dark-logo' />
+            <div className="top-bottom-div">
+
+                <div className="home-top-div">
+                    <div className="logo-div">
+                        <img src='/logo-dark.png' className='dark-logo' />
```
Comment (Nick-Yawn) [inline frontend/src/components/LandingPage/LandingPage.js:52]: I believe the source here should be changed to `static/logo-dark.png`, with `public/logo-dark.png` being moved to `public/static/logo-dark.png` — this is currently broken in production: https://the-jello-app.herokuapp.com/home

File: frontend/src/components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -3,20 +3,53 @@ import './LandingPage.css'
...
+                <div className='home-bottom-div'>
+                    <p><span id="bulk">Jell-O</span> is a variety of gelatin desserts, puddings, and no-bake cream pies</p>
+                    <p><span id="bulk">Trello</span> is a popular, simple-to-use collaboration tool that enables you to organize projects and everything related to them into boards</p>
+                    <p id="get-both-text">When you mix both... you get <span id="bulk">Jello</span>, a cuter Trello</p>
+                </div>
+                <div className="coder-profiles">
+
+                    <div className="coder">
+                        <a target="_blank" href="github.com">
+                            <img src="/jello9.png" className="coder-img jello-wiggle"></img>
```
Comment (Nick-Yawn) [inline frontend/src/components/LandingPage/LandingPage.js]: Similarly, I believe these files should be moved back into `/public/static/`, and the sources changed to match.

**Review by Nick-Yawn — CHANGES_REQUESTED** [review]
Looks excellent! I'm really stoked on how close it is to the wireframes!!!

File: frontend/src/components/LandingPage/LandingPage.js (hunk 1 above)
Comment (Nick-Yawn) [reply frontend/src/components/LandingPage/LandingPage.js:10]: Oh, and the `logo-dark.png` will also have to be moved into `static/`

**Review by MasonKogami — APPROVED** [review]

**Review by Nick-Yawn — APPROVED** [review]

**Pull request merged**